.github/stale.yml

```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
```

Comment:
Happy to remove these comments - or do you think they help explain?

Reply:
I think it's quite enough to understand the config w/o comments - properties are named good.

Reply:
Removing :)